engine/src/test/java/io/camunda/zeebe/engine/state/instance/JobStateTest.java

```diff
@@ -55,11 +55,12 @@ public void shouldCreateJobEntry() {
     jobState.create(key, jobRecord);
 
     // then
-    Assertions.assertThat(jobState.exists(key)).isTrue();
+    assertThat(jobState.exists(key)).isTrue();
     assertJobState(key, State.ACTIVATABLE);
     assertJobRecordIsEqualTo(jobState.getJob(key), jobRecord);
     assertListedAsActivatable(key, jobRecord.getTypeBuffer());
     refuteListedAsTimedOut(key, jobRecord.getDeadline() + 1);
+    refuteListedAsBackOff(key, jobRecord.getRecurringTime() + 1 + 1);
```

Comment:
:wrench: We could simplify the verification.

Do you write `jobRecord.getRecurringTime() + 1 + 1` by intention? 
Can we simplify it to `jobRecord.getRecurringTime() + 1` to align with the other verification?